composer.json

```diff
@@ -10,8 +10,8 @@
     "name": "ns8/protect-sdk",
     "require": {
         "php": "^7.1.3",
-        "zendframework/zend-http": ">=2.8.4 <=2.11",
-        "zendframework/zend-json": ">=2.6.1 <=3.1",
```

Comment:
I wonder if it'd be better to do something like ` <=2.*` and `<=3.*`, since major version bumps are expected to break backward compatibility, and I think if line 13 had `<=2.11.*` (or even looser `<=2.*`), we wouldn't have hit the issue.